pkg/mcproducer/mcproducer_test.go

```diff
@@ -35,6 +36,10 @@ var _ = Describe("ProduceMachineConfig", func() {
 		Expect(err).ToNot(HaveOccurred())
 		expectedRes, err := os.ReadFile("testdata/machineconfig-test.yaml")
 		Expect(err).ToNot(HaveOccurred())
+		if string(expectedRes) != res {
+			fmt.Printf("<%s>\n", res)
+			fmt.Printf("<%s>\n", expectedRes)
+		}
```

Comment:
This looks like debugging to me 🙂

Reply:
It was done intentionally: in case test fails, it would be easier to debug seeing the whole ouput of both MCO